### DISCUSSION TITLE
how to generate an echo top height plot

### DISCUSSION THREAD

**Discussion by MohitDahliya** [ARM-DOE/pyart#1221] [Q&A] [answered]
how can I calculate echo top height and plot it using pyart..?

**Comment by mgrover1** [comment, marked as answer]
You would need to do the following:
- Grid your radar ([gridding example](https://arm-doe.github.io/pyart/examples/mapping/plot_map_one_radar_to_grid.html#sphx-glr-examples-mapping-plot-map-one-radar-to-grid-py))
- Find the maximum height for a given reflectivity threshold

We do not have functions to do this directly.

**Comment by MohitDahliya** [comment]
Okay. Thanks. I'll try it and let you know if I face any issues.
